src/components/Notification/ToastNotification/ToastNotification.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from "react";
+import { Meta, StoryObj } from "@storybook/react";
+import { ToastNotificationProvider, useToastNotification } from "./index";
+import Button from "components/Button";
+
+const meta: Meta<typeof ToastNotificationProvider> = {
+  component: ToastNotificationProvider,
+  tags: ["autodocs"],
+};
+
+export default meta;
```

Comment:
This story is not using a height of 100vh for body and as such the floating notifications are rendered in the middle instead of on the bottom of the page

![image](https://github.com/user-attachments/assets/9ab9cc4d-68f0-4004-9812-64a5a94746ea)

it should ideally be looking like this

![image](https://github.com/user-attachments/assets/2841e30a-6682-4295-bd25-28c223edef8f)